library/options.inc.php

```diff
@@ -308,8 +308,10 @@ function parse_static_text($frow)
 
 // $frow is a row from the layout_options table.
 // $currvalue is the current value, if any, of the associated item.
+// $useFormatDate - change values of dates inputs to local format date (according $GLOBALS['date_display_format']).
+// IMPORTANT - in the case $useFormatDate is true, not forget to convert a dates back to mysql format using DateToYYYYMMDD() function before saving.
 //
-function generate_form_field($frow, $currvalue)
+function generate_form_field($frow, $currvalue, $useFormatDate = false)
```

Comment:
TODO
Instinct tells me that will be best to simply get it working in all layout based structure and not need this parameter (I will look into this)

Reply:
Removed this parameter and always used in layout engine.